cpp/test/Ice/span/TestAMD.ice

```diff
@@ -10,34 +10,36 @@ module Test
     sequence<short> ShortSeq;
     sequence<string> StringSeq;
 
+    // No metadata on in-parameters are we don't use this generated code for the client.
```

Comment:
```suggestion
    // No metadata on in-parameters as we don't use this generated code for the client.
```
This sentence needs a tweak.

But I also think we should just keep this metadata here.
There is value to testing that applying this metadata to in-parameters has no effect on the server side.

And we usually keep `Test.ice` and `TestAMD.ice` in close sync with one another (except for `["amd"]` of course).
introducing inconsistencies will make them harder to maintain IMO.